index.bs

```diff
@@ -1142,8 +1152,9 @@ The attestation data encodes contextual bindings made by the <a>authenticator</a
 data has a compact but extensible encoding. This is desired since authenticators can be devices with limited capabilities and
 low power requirements, with much simpler software stacks than the client platform components.
 
-The field `rawData` for this type is a byte array of 45 bytes + length of public key + length of KeyHandle + potentially more 
-extensions. The first bytes before the extensions have a fixed layout as follows: 
+The field `rawData` for this type is a byte array of 17 bytes + length of AAGUID + length of public key + length of Credential
+ID + length of clientDataHash + potentially more extensions. The first bytes before the extensions have a fixed layout as
+follows:
```

Comment:
I think I (finally) figured out how we keep getting various different values for the fixed length of the rawData before the public key itself -- it is because some of us (i.e. myself) take the `17 bytes` (or the old `45 bytes`) to mean the length of rawData before the *first* variable length field, rather than the *sum of all of the fixed-length rawData fields*, which it seems is (now more clearly) stated above. 

Thus I think the above paragraph is needlessly misleading and we should update it.  Here is a proposal..
> The packed attestation `rawData` field is a variable-length byte array, consisting of various fixed-length and variable-length sub-fields, arranged as given in the below table: